src/modules/tools/zprobe/CalibrationStrategy.h

```diff
@@ -0,0 +1,64 @@
+// Generic calibration strategy based on nonlinear optimization
+//
+// Copyright (c) Oskar Linde 2015
+//
+// Licence: GPL v2
+
+#ifndef _GENERICLEVELINGSTRATEGY
+#define _GENERICLEVELINGSTRATEGY
+
+#include "LevelingStrategy.h"
+#include <string>
+#include <vector>
+
+#define calibration_strategy_checksum CHECKSUM("calibration")
+struct V3 { float m[3]; };
```

Comment:
Agreed. I actually wanted to use Vector3 here, but it doesn't provide public access to its m member. Maybe something to consider changing? Directly editing the m value in a Vector3 will always be safe.